resources/metadata/kanikoExecute.yaml

```diff
@@ -1,7 +1,23 @@
 metadata:
   name: kanikoExecute
   description: Executes a [Kaniko](https://github.com/GoogleContainerTools/kaniko) build for creating a Docker container.
-  longDescription: Executes a [Kaniko](https://github.com/GoogleContainerTools/kaniko) build for creating a Docker container.
+  longDescription: |
+    Executes a [Kaniko](https://github.com/GoogleContainerTools/kaniko) build for creating a Docker container.
+
+    ### Building multiple container images
+
+    The step allows you to build multiple container images with one run.
+    This is suitable in case you need to create multiple images for one microservice, e.g. for testing.
+
+    All images will get the same "root" name and the same versioning.<br />
+    **Thus, this is not suitable to be used for a monorepo approach!** For monorepos you need to use a build tool natively capable to take care for monorepos 
+    or implement a custom logic and for example execute this `kanikoExecute` step multiple times in your custom pipeline.
```

Comment:
Isn't this limitation mainly artificial? I.e. when we would provide a way to handle the naming (and versioning) part (e.g. via parameters), we could easily allow multiple distinct images to be produced.

Reply:
kind of, it takes the status quo into consideration for sure.
I don't want to open monorepo discussions here but I see a need to make this explicit.

Reply:
Well, I tend to agree that putting multiple microservices into a single repository kind of defeats the purpose of splitting it into mircoservices.

However, I am not sure if a CI/CD library is the right place to educate developers by artificially introducing (or keeping up) limitations.